From: Jan-Ivar Bruaroey <jan-ivar@users.noreply.github.com>
Date: Fri, 9 Feb 2024 15:35:24 -0500
Subject: (tmp-cherry-pick) Revert "Add RTC_EXPORT for Voucher::ScopedSetter
 and Voucher::Attachment." (dd45ab650e)

This reverts commit 414ffac4b742dc3a1696deb3117ef1383fc8d9be.

Reason for revert: remove functionality after measurement complete

Original change's description:
> Add RTC_EXPORT for Voucher::ScopedSetter and Voucher::Attachment.
>
> Bug: chromium:1498378
> Change-Id: I606079411c71ac46cc5d261a86f6a11c473d1278
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/325960
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#41071}

Bug: chromium:1498378
Change-Id: If35641719a8bf01b045e10c707af809250665330
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/327022
Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Reviewed-by: Harald Alvestrand <hta@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#41130}
---
 rtc_base/voucher.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rtc_base/voucher.h b/rtc_base/voucher.h
index 1a82551b74..30a6460906 100644
--- a/rtc_base/voucher.h
+++ b/rtc_base/voucher.h
@@ -32,7 +32,7 @@ class RTC_EXPORT Voucher {
 
   // Vouchers aggregate attachments, which are application-specific attachments
   // that have logic unrelated to the mechanics of Voucher progression.
-  class RTC_EXPORT Attachment {
+  class Attachment {
    public:
     using Id = size_t;
 
@@ -45,7 +45,7 @@ class RTC_EXPORT Voucher {
 
   // Scoped setter that saves the current voucher on stack and instates a new
   // one, until the scope exits.
-  class RTC_EXPORT ScopedSetter {
+  class ScopedSetter {
    public:
     explicit ScopedSetter(Ptr voucher);
     ~ScopedSetter();
